week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,19 +20,36 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    while (this.volume < 100) {
+
+      this.volume++;
+    }

```

Comment:
Review the requirements again. 
It says the fillUp will fill it up with no checks!